router/middleware/perm/perm.go

```diff
@@ -87,21 +87,38 @@ func MustSecretAdmin() gin.HandlerFunc {
 				return
 			}
 		case constants.SecretShared:
-			logrus.Debugf("Verifying user %s has 'admin' permissions for team %s/%s", u.GetName(), o, n)
+			if n == "*" {
+				logrus.Debugf("Verifying user %s has 'admin' permissions for org %s", u.GetName(), o)
 
-			perm, err := source.FromContext(c).TeamAccess(u, o, n)
-			if err != nil {
-				logrus.Errorf("unable to get user %s access level for team %s/%s: %v", u.GetName(), o, n, err)
-			}
+				perm, err := source.FromContext(c).OrgAccess(u, o)
+				if err != nil {
+					logrus.Errorf("unable to get user %s access level for org %s: %v", u.GetName(), o, err)
+				}
 
-			if !strings.EqualFold(perm, "admin") {
-				// nolint: lll // ignore long line length due to error message
-				retErr := fmt.Errorf("user %s does not have 'admin' permissions for the team %s/%s", u.GetName(), o, n)
+				if !strings.EqualFold(perm, "admin") {
+					retErr := fmt.Errorf("user %s does not have 'admin' permissions for the org %s", u.GetName(), o)
```

Comment:
Could we add a `nolint` comment based off the linter feedback?

https://github.com/go-vela/server/actions/runs/1099137662

```suggestion
					// nolint: lll // ignore long line length due to error message
					retErr := fmt.Errorf("user %s does not have 'admin' permissions for the org %s", u.GetName(), o)
```